### PR TITLE
docs: promotional scheme and pricing rule

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
## Updated Pricing Rule Screenshots
Screen Shot 2019-06-14 at 4 34 36 pm

## Added docs for the Promotional Schemes
Screen Shot 2019-06-14 at 4 35 36 pm

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md [hunk 1]
```diff
@@ -5,34 +5,41 @@ Pricing Rule is a master where you can define rules based on which discount is a
...
-### 1. How to create a Pricing Rule
+## 1. How to create a Pricing Rule
 1. Go to: **Selling > Items and Pricing > Pricing Rule > New**.
 1. Set a title for the rule.
 1. Select what to Apply On.
+1. Select whether you want to apply price dicount or product discount. If you want to give free products then select the product discount.
 
-    <img alt="Applicable On" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-on.png">
-1. For a single item, select Item Code and select the item.
+<img alt="Applicable On" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-on.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md]: crop needed on the top, a black line is visible

File: erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md [hunk 2]
```diff
@@ -5,34 +5,41 @@ Pricing Rule is a master where you can define rules based on which discount is a
...
+
+1. For a single item, select Item Code and select the items.
 1. If you want Pricing Rule to be applied on all the items, select 'Item Group' and select **All Item Group** (parent Item Group).
 1. If you have multiple price lists, you can set the priority from 0 to 20.
 1. Set whether the Pricing Rule is for Selling of Buying the item.
 1. Set what type is this applicable for and select the record. You can set applicability to one of the following masters.
 
-    <img alt="Applicable for" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-for.png">
+<img alt="Applicable for" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-for.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md]: highlight the part or crop to show only the applicable types

File: erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md [hunk 3]
```diff
@@ -44,31 +51,46 @@ To have Discount and Price List Rate for an Item auto-applied, create Pricing Ru
...
 
-### 3. Scenarios with Pricing Rule
-#### 3.1 Make Price List
+1. Discount can be set in terms of amount.
+
+    <img alt="Discount" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-discount-amt.png">
+
+### 2.2 Product Discount
+
+1. Buy 2 quantities get 1 free quantity of the same item, to configure such type of rules. Set the Price or Product Discount as Product discount and enable Same Item and set the quantity.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md]: "Buy 2 quantities get 1 free quantity of the same item." To configure such type of rules, set the Price or Product Discount as Product discount, tick the Same Item checkbox, and set the quantity.

File: erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md [hunk 4]
```diff
@@ -44,31 +51,46 @@ To have Discount and Price List Rate for an Item auto-applied, create Pricing Ru
...
+
+    <img alt="Discount" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-same-product-free.png">
+
+1. Buy 2 quantities get 1 free quantity of the another item, to configure such type of rules. Set the Price or Product Discount as Product discount, disable Same Item and set the Free Item and quantity.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md]: "Buy 2 quantities get 1 free quantity of the another item." To configure such type of rules. Set the Price or Product Discount as Product discount, untick the Same Item checkbox and set the Free Item and quantity.

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+# Promotional Scheme
+
+Promotional schemes are the key to successful retail stores. Promotional schemes can be easily configure in the erpnext. Promotional scheme is linked to the pricing rule, against each slab system will generate the pricing rule.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: Promotional schemes help retail stores become successful. They can be easily configure in ERPNnext. A Promotional scheme is linked to a pricing rule, against each slab system will generate the pricing rule.

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+# 1. How to create a Promotional Scheme
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: H2 !!!
correct levels in following headings also

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+# Promotional Scheme
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: is this a v12 feature?

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+Promotional schemes are the key to successful retail stores. Promotional schemes can be easily configure in the erpnext. Promotional scheme is linked to the pricing rule, against each slab system will generate the pricing rule.
+
+# 1. How to create a Promotional Scheme
+
+1. Go to: **Selling > Items and Pricing > Promotional Scheme > New**.
+1. Set a title for the rule.
+1. Select what to Apply On like Item Code, Item Group, Brand etc.
+1. Based on the apply on system will give you the provision to select the Item Code / Item Group / Brand in the table.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: Based on the 'Apply On', system will give you the provision to select the Item Code / Item Group / Brand in the table.

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+1. Then configured the party information.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: configure
configured is past tense

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+1. Set promotional schemes for price dicsount, product discount.
+1. User can also apply the discount on the other item / item group / brand by selcting the value for Apply Rule On Other field.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: first letter caps for Item, Item Group, Brand

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes.png">
+
+## 1.1 Price Discount
+
+In this type of promotional schemes, the user gets an option to set the discount in terms of percentage or amount based on the min quantity, max quantity, min amount and max amount on the products. User can also configure the scheme where they can set the flat rate for the product based on the quantity or the amount of the product.
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes-price-discount.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: needs cropping at the bottom

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+## 1.2 Product Discount
+
+In this type of promotional schemes, the user gets an option to give the free product on purchase of the same or different product with conditions like min quantity, max quantity, min amount, max amount.
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes-product-discount.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: needs cropping top and bottom

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+# 2. How to configure a Promotional Scheme
+
+Lets understand how to configure a promotional scheme in erpnext using some examples.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: ERPNext

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+
+## 2.1 Mixed Conditions Schemes
+
+Customer A has purchased 10 quantities of Britannia Cake 5 Rs packet and 5 quantities of Britannia Cake 10 Rs packet. Now shopkeeper wants to give the discount of 10% to the customer A. This shopkeeper also wants to give 10 % discount to the customer B who has purchased 15 quantities of Britannia Cake 5 Rs packet. So basically user wants to apply the discount on the products Britannia Cake 5 Rs, Britannia Cake 10 Rs only if his clients have purchased the 15 quantities of any product or sum of both product.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: customer A,,,,,, customer B (remove 'the')
remove the: the products Britannia Cake 5 Rs
remove the before 15 quantities

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+
+To configure this in erpnext the stpes are as follows:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: ERPNext

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+1. Set Apply On as Item Code.
+1. Set the item code Britannia Cake 5 Rs, Britannia Cake 10 Rs in the Pricing Rule Item Code table.
+1. Enable "Mixed Conditions" field
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: fullstop(.) end your sentences

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+1. In price discount table, set the min qty, max qty as 15
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: .

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+1. Set the discount type as Discount Percentage and rate as 10.
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes-mixed-conditions.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: tab before image tag
also crop screenshot if bottom parts are not needed

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+
+## 2.2 To apply discount on other item
+
+Customer A has purchased 30 quantities of Britannia Cake 5 Rs packet and 2 quantities of the product Britannia Cake 15 Rs then the user wants to sell the product Britannia Cake 15 Rs at the flat rate 12. Here the original price for the product Britannia Cake 15 Rs is 15.
+The user wants to apply the rule only if the customer has purchased min 30 quantities of the product Britannia Cake 5 Rs or Britannia Cake 10 Rs.
+
+To configure this in erpnext the stpes are as follows:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: ERPNext

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+1. Set Apply On as Item Code.
+1. Set the item code Britannia Cake 5 Rs, Britannia Cake 10 Rs in the Pricing Rule Item Code table.
+1. Apply Rule On Other as Item Code and set Item Code as Britannia Cake 15 Rs
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: .

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md [hunk 21]
```diff
@@ -0,0 +1,64 @@
...
+1. In price discount table, set the min qty as 30.
+1. Set the discount type as Rate and rate as 12.
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes-discount-on-other.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: tab before image tag
also crop screenshot if bottom parts are not needed

**Comment by rohitwaghchaure** [comment]
@prasadarr Changes are done

File: erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md (hunk 7 above)
Comment (rohitwaghchaure) [reply erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md]: Yes

**Pull request merged**